acceptance-tests/src/test/java/uk/gov/di/test/pages/BasePage.java

```diff
@@ -1,24 +1,18 @@
 package uk.gov.di.test.pages;
 
 import io.cucumber.java.Scenario;
-import org.openqa.selenium.By;
-import org.openqa.selenium.Cookie;
-import org.openqa.selenium.JavascriptExecutor;
-import org.openqa.selenium.OutputType;
-import org.openqa.selenium.TakesScreenshot;
-import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.WebElement;
+import org.openqa.selenium.*;
```

Comment:
Think your editor has probably auto wildcard imported here, we try and avoid this as per https://gds-way.digital.cabinet-office.gov.uk/manuals/programming-languages/java.html#imports

Reply:
roger dodger, i'll sort it!